include/clspv/Compiler.h

```diff
@@ -59,6 +67,14 @@ typedef enum ClspvError {
 #define EXPORT __attribute__((visibility("default")))
 #endif
 
+#ifndef clspv_restrict
+
+/** Optional restrict qualifier for given C API with C++ implementation where
+ * restrict is not the thing. */
+#define clspv_restrict
+
+#endif
+
```

Comment:
I don't like that optional qualifier on a public interface. Either we want it or not, but I don't think making it optional is a good idea.
As that interface already exists it better not to change it anymore.
Please remove